app/models/edition.rb

```diff
@@ -247,8 +247,8 @@ def self.latest_edition
     joins(:document).where("editions.id = documents.latest_edition_id")
   end
 
-  def self.latest_published_edition
-    joins(:document).where("editions.id = documents.live_edition_id AND state = 'published'")
+  def self.latest_live_edition
```

Comment:
So if we want to retain the current behaviour the correct thing to do here is update this to:

```
joins(:document).where("editions.id = documents.latest_edition_id and state = 'published'")
```

Since all our production code is scoped to a single document there might even be an argument for removing this?